compiler/optimizer/SinkStores.cpp

```diff
@@ -2320,250 +2261,6 @@ TR_SinkStores::replaceLoadsWithTempSym(TR::Node *newNode, TR::Node *origNode, TR
    }
 
 
-// A single store that is being replicated to a side exit may have 0 or more nodes replaced with anchored nodes for different reasons
-// insertAnchoredNodes will take care of all possible node replacements so only a single recursive walk of a store is required
-uint32_t
-TR_SinkStores::insertAnchoredNodes(TR_MovableStore *store,
```

Comment:
Note to reviewers:  this function is unreachable (except via a recursive call).  It is removed as well as the data structures that only it used.  Most of the other deletions in this PR are derived from this removal.